.github/workflows/build-and-test.yaml

```diff
@@ -26,13 +29,18 @@ jobs:
       - name: Install dependencies
         run: bundle install
 
+      - name: Start MongoDB
```

Comment:
How does mongodb tests works?

Reply:
It's not `mongodb tests`. We have tests for our app, the latter which uses MongoDB. This step of the workflow only exists to run the database needed in the tests and is needed because we are not using sqlite, which Rails would run by default